### ISSUE TITLE
Will LLaVA-NeXT-interleave 0.5B release?

### ISSUE THREAD

**Issue by zihaolucky** [LLaVA-VL/LLaVA-NeXT#74]
I think the 0.5B model already shown strong performance in Multi-image Evaluation, will this model release later? Thanks.

**Comment by FengLi-ust** [comment]
Yes, we have released the 0.5b model [here](https://huggingface.co/lmms-lab/llava-next-interleave-0.5b)
